examples/persistence_service/persistent_storage/c++11/README.md

```diff
@@ -33,71 +41,115 @@ Makefiles in the configuration process, run make to build the example. Likewise,
+    ./hello_world_publisher -domain_id 70 -persistent 0 -sample_count 10
     ```
 
--   Persistence Service Scenario
+    **Note**: The `-sample_count` argument controls how many samples the publisher
+    will publish.
+
+    If everything goes well you should see:
 
     ```sh
-    # Do not forget to replace the path separator to "\" and
-    # "$NDDSHOME" by "%NDDSHOME%" on Windows.
+    Writing hello_world, count 0
+    Writing hello_world, count 1
+    Writing hello_world, count 2
+    Writing hello_world, count 3
+    Writing hello_world, count 4
+    ```
 
-    # Run persistence service (in the same folder that there are
-    # persistence_service_configuration.xml):
-    "$NDDSHOME/bin/rtipersistenceservice" -cfgFile persistence_service_configuration.xml -cfgName <persistence_service_database|persistence_service_filesystem>
+3.  Run the subscriber from the `build` directory you created when compiling the
+    example. The publisher and subscriber have the `-domain_id` and `-persistent`
+    arguments to control the domain ID and the type of *Persistence Service*
+    scenario you are running.
 
-    hello_world_subscriber
-    hello_world_publisher -sample_count 5 -initial_value 0
-    hello_world_subscriber
+    For this step we use the `defaultTransient` configuration while running on
+    domain ID 70.
+
+    ```sh
+    ./hello_world_subscriber -domain_id 70 -persistent 0
     ```
 
-The applications accepts different arguments:
+    If everything goes well you should see:
 
-```plaintext
-hello_world_subscriber:
-    -domainId <domain ID> (default: 0)
-    -sample_count <sample_count> (default: infinite => 0)
-    -drs <1|0> Enable/Disable durable reader state (default: 0)
+    ```sh
+    [data: 0]
+    [data: 1]
+    [data: 2]
+    [data: 3]
+    [data: 4]
+    ```
 
-hello_world_publisher:
-    -domainId <domain ID> (default: 0)
-    -sample_count <number of published samples> (default: infinite)
-    -initial_value <first sample value> (default: 0)
-    -sleep <sleep time in seconds before finishing> (default: 0)
-    -dwh <1|0> Enable/Disable durable writer history (default: 0)
-```
+4.  Stop the publisher by pressing CTRL-C.
+
+5.  Start a new subscriber and verify that it receives samples from *Persistence
+    Service* in the absence of the original publisher.
+
+    ```sh
+    ./hello_world_subscriber -domain_id 70 -persistent 0
+    ```
+
+    If everything goes well you should see:
+
+    ```sh
+    [data: 0]
+    [data: 1]
+    [data: 2]
+    [data: 3]
+    [data: 4]
+    ```
+
+6.  Stop RTI Persistence Service and the two subscribers by pressing CTRL-C.
+
+7.  Restart RTI Persistence Service. This time we use the `defaultPersistent`
+    configuration to persist data to the disk.
+
+    ```sh
+    $NDDSHOME/bin/rtipersistenceservice -cfgName defaultPersistent
+    -cfgFile PersistenceServiceConfig.xml -domainId 70
+    ```
+
+8.  Repeat steps 2 to 6 but this time use `-persistent 1` when starting the
```

Comment:
This example is missing the these steps from the old example which show you can restart persistence service and it loads its databases (the data is not loss)

Stop RTI Persistence Service and the two subscribers by pressing CTRL-C.

Restart RTI Persistence Service.

${NDDSHOME}/bin/rtipersistenceservice -cfgFile PsHelloWorld.xml
-cfgName <HelloWorldMySQL|HelloWorldFile>
RTI Persistence Service started

Start a subscriber and verify that it receives the persisted samples from
RTI Persistence Service:

objs/i86Linux2.6gcc3.4.3/HelloWorld_subscriber [domain_id]

The output will look something like this:
HelloWorld subscriber sleeping for 4 sec...
data: 0
data: 1
data: 2
data: 3
data: 4

Reply:
I agree. I missed out on that part. Done.